packages/ui/src/ui-component/dropdown/AsyncDropdown.js

```diff
@@ -105,7 +105,11 @@ export const AsyncDropdown = ({
         })()
 
         // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [])
+    }, [credentialNames])
+
+    useEffect(() => {
```

Comment:
this triggers inifite loop

Reply:
I've reverted this change. I added this because the component wasn't rerendering on prop change. But it turns out the `useEffect` is not needed. It works without it.